src/inmanta/loader.py

```diff
@@ -186,6 +154,44 @@ def get_file_hashes(self) -> Iterable[str]:
         """Return the hashes of all source files"""
         return (info.hash for info in self.__file_info.values())
 
+    def get_module_source_info(self) -> dict[str, list["SourceInfo"]]:
+        """Return all module source info"""
+        return self.__module_to_source_info
+
+    def get_module_version_info(self) -> dict[str, "PythonModule"]:
+        """Return all module version info"""
+        if self.__modules_data:
+            return self.__modules_data
+
+        source_info = self.get_module_source_info()
+
+        modules_data = {}
+        for module_name, files_in_module in source_info.items():
```

Comment:
Why not use this code in the database migration? i.e. the has calculation is now in two places (it will works, just not very efficiently)